src/app/faucet/page.tsx

```diff
@@ -0,0 +1,9 @@
+import FaucetPage from "./client-page";
```

Comment:
Instead of the faucet page being a client component in its entirety, we should try to keep as much of it as possible as a server component. We can keep all of the components needed in `client-page.tsx` into a single file, this is just to avoid the confusion of having 2 files with "page" in it to cover 1 route. 

Eg:

Lets move the layout of the page into here, and `ClaimCoins` and `ConnectWalletModal` can be client components in a different file. 

```
export default function Faucet() {
  return (
    <>
      <Head>
        <meta name="description" content="Generated by create next app" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <link rel="icon" href="/favicon.ico" />
      </Head>
      <Container>
        <Flex align="center">
          <Heading color="white" fontSize="20px">
            Thala Faucet: Claim test coins
          </Heading>
          <Spacer />
          <ConnectWalletModal />
        </Flex>
        <ClaimCoins />
      </Container>
    </>
  );
}
```

Reply:
Can you just do the layout here and rename the file to `faucet-components` just to avoid confusion.